test/util/display_SYSINFO_settings.py

```diff
@@ -1,31 +1,28 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
+1,
@gpaulos  by the way, this file name has a uppercase 'SYSINFO' which is not consistent with other file names unless it has to, at present, on README, Dockerfile has uppercase in file name.